client.py

```diff
@@ -10,10 +11,11 @@
 """
 
 logging.basicConfig(level=logging.INFO)
-
+HazelcastCloudDiscovery._CLOUD_URL_BASE = "YOUR_DISCOVERY_URL"
```

Comment:
Will this (and its import) be visible to the user? Personally, I think that users should not see such internal details

Reply:
No, It won't be visible in Production